script/create-superuser

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/php
+
+<?php
+
+// create-superuser.php
+
+// This script creates a superuser account in the database.
+// It is intended to be be used to create a test superuser in dev
+
+// Running the file will prompt for a username and password
+// or these can be passed in with -u and -p arguments.
+
+include_once dirname(__FILE__) . "/../www/includes/easyparliament/init.php";
+
+// Parse command line arguments
+$options = getopt("u:p:");
+
+// Check if username and password were passed
+if (isset($options["u"]) && isset($options["p"])) {
+    $email = $options["u"];
+    $password = $options["p"];
+    // if username isn't set, prompt for it
+}
+if (!isset($options["u"])) {
+    echo "Enter username (email address): ";
+    $email = trim(fgets(STDIN));
+    // if password isn't set, prompt for it
+}
+
+if (!isset($options["p"])) {
+    echo "Enter password: ";
+    system("stty -echo");
+    $password = trim(fgets(STDIN));
+    system("stty echo");
+}
+
+$details = [
+    "firstname" => "Test Admin",
+    "lastname" => "User",
+    "email" => $email,
+    "postcode" => "N13 4BS",
+    "url" => "https://mysociety.org",
+    "password" => $password,
+    "optin" => 1,
+    "status" => "Administrator",
+    "confirmed" => 1,
+];
+
+$registrationtime = gmdate("YmdHis");
+$passwordforDB = password_hash($details["password"], PASSWORD_BCRYPT);
+
+# get database connection
+$db = new ParlDB();
+
+$q = $db->query(
```

Comment:
Rather than this whole query, could we use the user->add($details, $confirmation_required=false) function that exists already? Looks like you pass in a hash of various entries, but it does some for you (e.g. password hash).

Reply:
This probably *can* be done, but I was having trouble getting useful errors from why it wasn't working for me, so I gave up and brought the relevant code here till it was working.